link/executable.go

```diff
@@ -0,0 +1,78 @@
+package link
```

Comment:
How do you feel about merging executable.go with uprobe.go? As it is right now `type executable` is split across files, which I find hard to follow.

Reply:
👍🏻